src/docs/deployment/android.md

```diff
@@ -273,6 +276,9 @@ the Play Store.
 This section describes how to build a release app bundle.
 If you completed the signing steps,
 the app bundle will be signed.
+At this point, you might consider [obfuscating your Dart code][]
+to make it more difficult to reverse engineer. Obfuscating
+your code involves adding a couple flags to your build command.
```

Comment:
Perhaps: "..and maintaining additional files to de-obfuscate stack traces"? or something to that nature.